src/container/EdgeRenderer/index.tsx

```diff
@@ -218,49 +222,156 @@ const EdgeRenderer = (props: EdgeRendererProps) => {
     connectionLineComponent,
     onlyRenderVisibleElements,
   } = props;
-  const transformStyle = `translate(${transform[0]},${transform[1]}) scale(${transform[2]})`;
```

Comment:
EdgeRenderer already inherits the transformStyle now that it's nested in the NodeRenderer, so if we kept this transformStyle you'd see a bad transformation when zooming in and out